test/unit/behaviors/cache_store_format_version_behavior.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module CacheStoreFormatVersionBehavior
```

Comment:
There look to be some more tests in https://github.com/rails/rails/blob/main/activesupport/test/cache/behaviors/cache_store_format_version_behavior.rb. Could we pull those in as well?

Reply:
https://github.com/rails/solid_cache/pull/104/commits/2c578d7e364332433dcf7647a89adb66fd6c1343